sem-service-check-params

```diff
@@ -86,6 +86,43 @@ params-check::postgres () {
   fi
   echo "$service_version $default_params"
 }
+params-check::postgis () {
+  default_params='--net=host --rm -d -e POSTGRES_PASSWORD=semaphore --name postgis -v /var/tmp/postgres:/var/lib/postgresql/data -v /var/run/postgresql:/var/run/postgresql'
+  service_version="9.6-2.5"
+  sudo rm -rf /var/run/postgresql /var/tmp/postgres
+  sudo mkdir -p /var/run/postgresql ; sudo chmod -R 0777 /var/run/postgresql
+  sudo mkdir -p /var/tmp/postgres ; sudo chmod -R 0777 /var/tmp/postgres
```

Comment:
Everything is logical except this part. Why are we deleting there folders?

Reply:
The bats test revealed that starting postgres 11 fails when postgres 9 db files are in place.